.editorconfig

```diff
@@ -8,3 +8,8 @@ end_of_line = lf
 charset = UTF-8
 trim_trailing_whitespace = false
 insert_final_newline = true
+
+
+[package.json]
+indent_style = space
```

Comment:
@dsp Why spaces? If Prettier uses tabs, and I hope it still does, this will confuse the GitHub code viewer.